src/Utility/Assembly.cpp

```diff
@@ -107,9 +108,9 @@ void Assembly::writeInstruction(const llvm::MCInst inst, memory_ostream *stream)
 }
 
 
-void Assembly::printDisasm(const llvm::MCInst &inst, llvm::raw_ostream &out) const {
+void Assembly::printDisasm(const llvm::MCInst &inst, uint64_t address, llvm::raw_ostream &out) const {
```

Comment:
``uintptr_t`` instead of of ``uint64_t``

Reply:
I prefer to keep a ``uint64_t`` to match the LLVM API (https://github.com/llvm/llvm-project/blob/llvmorg-10.0.1/llvm/include/llvm/MC/MCDisassembler/MCDisassembler.h#L77)